### DISCUSSION TITLE
Is there a way to run more than 10,000 timers in a test?

### DISCUSSION THREAD

**Discussion by justinfarrelldev** [vitest-dev/vitest#4925] [Q&A] [answered]
I am using Vitest for some algo-trading backtesting and I ran into this: 
```
Error: Aborting after running 10000 timers, assuming an infinite loop!
```
The thing is - it's not an infinite loop, I'm just using a `setInterval` set to run every 2 minutes over a period of a month (and I'm using the fake timers to simulate that behavior). 

Is there a way to set this timer limit higher? I googled this and, unsurprisingly, there were very few answers (presumably because 99.9% of the time, this error only shows when there was a very obvious infinite loop).

Thank you!

**Comment by sheremet-va** [comment, marked as answer]
There is a config option: https://vitest.dev/config/#faketimers-looplimit

```ts
export default defineConfig({
  test: {
    fakeTimers: {
      loopLimit: 100_000,
    }
  }
})
```

**Reply by justinfarrelldev** [reply]
Thank you!
